R/pkg/R/mllib_regression.R

```diff
@@ -110,7 +125,8 @@ setClass("IsotonicRegressionModel", representation(jobj = "jobj"))
 #' @seealso \link{glm}, \link{read.ml}
 setMethod("spark.glm", signature(data = "SparkDataFrame", formula = "formula"),
           function(data, formula, family = gaussian, tol = 1e-6, maxIter = 25, weightCol = NULL,
-                   regParam = 0.0, var.power = 0.0, link.power = 1.0 - var.power) {
+                   regParam = 0.0, var.power = 0.0, link.power = 1.0 - var.power,
+                   stringIndexerOrderType = "frequencyDesc") {
```

Comment:
given we need to spell it out, I'm wondering it it would be better to check the `stringIndexerOrderType` parameter to match on of the supported options in R?